<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Titteli</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Push</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTitteli">
    <property name="title">
     <string>Foo</string>
    </property>
    <addaction name="actionQuit"/>
    <addaction name="actionKalatus"/>
    <addaction name="actionKulutus"/>
   </widget>
   <widget class="QMenu" name="menuBarri">
    <property name="title">
     <string>Bar</string>
    </property>
    <addaction name="actionBorri"/>
    <addaction name="actionBurri"/>
   </widget>
   <addaction name="menuTitteli"/>
   <addaction name="menuBarri"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
   <addaction name="actionKalatus"/>
   <addaction name="actionBorri"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionKalatus">
   <property name="text">
    <string>Menu2</string>
   </property>
  </action>
  <action name="actionBorri">
   <property name="text">
    <string>Bar1</string>
   </property>
  </action>
  <action name="actionBurri">
   <property name="text">
    <string>Bar2</string>
   </property>
  </action>
  <action name="actionKulutus">
   <property name="text">
    <string>Menu3</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
